### ISSUE TITLE
When a series marked as "Planning" changes from 0 to 1 episodes watched, ask the user if they'd like to change the status to "Watching"

### ISSUE THREAD

**Issue by CBozsik** [lotusprey/otraku#171]
Currently in the list, hitting the +1 episode button only adds an episode to the count, keeping it in the Planning list.

It would be nice if it also prompted to move the series to Watching to avoid having to go through the menus to do that separately.

**Comment by lotusprey** [comment]
When you increment the counter in the sheet, the status and start date _will_ change and a snackbar will appear to warn you.
I'm not sure if I want to have this behavior directly in the lists, because when the list updates, the entry will get moved to another list and it will disappear from view, making this change hard to reverse. In the case of the sheet, the changes aren't committed until you press "Save", so it seems more manageable to me.

**Comment by Mistblown** [comment]
Yeah, that's how it reacts in [Al-chan](https://play.google.com/store/apps/details?id=com.zen.alchan) for example. Disappearing from the 'Planning' view shortly after approving the move. (Video example attached).

I thought it made sense from a User standpoint to offer the option to move it, and approving the move makes sense that it would disappear from the current 'planned' view.

https://github.com/user-attachments/assets/9250dfc9-80a8-44f0-8ad1-02a1aea4cfde

**Comment by lotusprey** [comment]
Ah, it asks you whether to move it. Ok, that makes sense, I might add it.
